pkgs/by-name/aw/awf/package.nix

```diff
@@ -29,23 +32,54 @@ stdenv.mkDerivation rec {
   buildInputs = [
     gtk2
     gtk3
+    gtk4
   ];
 
+  configureFlags = [ "--disable-gtk5" ];
```

Comment:
~~gtk5 doesn't exist yet.~~ [they do have an option for it somehow](https://github.com/luigifab/awf-extended/blob/8dea93f78f04f570acf498766d88aa42a34593a1/configure.ac#L16)

Reply:
Sorry, I noticed you were the dev behind `awf-extended`. Not sure whether or not this should go in nixpkgs though...

Reply:
gtk5 isn't yet released, but I already prepared configure/make and packages for gtk5 when I added all configure options